lib/event_loop.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const Loopbench = require('loopbench');
+let sampler = null;
+
+
+module.exports = { start, stop, getState };
+
+
+function start (options) {
+  if (sampler) {
+    return;
+  }
+
+  sampler = Loopbench(options);
```

Comment:
Right now I'm supporting changing the sampling interval in loopbench by stopping and then starting again. If you support dynamically changing it in loopbench, it would take my code for that from 5 lines to 1, but it doesn't have to be a high priority for you.